src/Honeybee_FileExplorer.py

```diff
@@ -0,0 +1,49 @@
+#Developed by: Sarith Subramniam, Penn State
```

Comment:
License should follow Honeybee's licencing pattern. Here is an example: https://github.com/mostaphaRoudsari/Honeybee/blob/9614dd7894feeb459fcd2d97f1ab7945051809c0/src/Honeybee_Construct%20Energy%20Balance.py#L1-L20